src/Microsoft.Health.Fhir.Core/Features/Security/Authorization/SMARTScopeFhirAuthorizationService.cs

```diff
@@ -0,0 +1,37 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Core.Features.Context;
+using Microsoft.Health.Fhir.Core.Features.Context;
+
+namespace Microsoft.Health.Fhir.Core.Features.Security.Authorization
+{
+    /// <summary>
+    /// determines access based on the SMART scope.
+    /// </summary>
+    public static class SMARTScopeFhirAuthorizationService
+    {
+        public static DataActions CheckSMARTScopeAccess(RequestContextAccessor<IFhirRequestContext> requestContextAccessor, DataActions dataActions)
```

Comment:
Public static here means it would be a good candidate for an extension method

```suggestion
        public static DataActions CheckSMARTScopeAccess(this RequestContextAccessor<IFhirRequestContext> requestContextAccessor, DataActions dataActions)
```